Cake.Recipe/Content/build.cake

```diff
@@ -132,10 +127,7 @@ BuildParameters.Tasks.DotNetCoreRestoreTask = Task("DotNetCore-Restore")
 {
     DotNetCoreRestore(BuildParameters.SolutionFilePath.FullPath, new DotNetCoreRestoreSettings
     {
-        Sources = new [] {
-            "https://api.nuget.org/v3/index.json",
-            "https://www.myget.org/F/cake-contrib/api/v3/index.json"
-        },
+        Sources = BuildParameters.NuGetSources,
```

Comment:
Ah I see, good catch I'll fix, was blind and didn't see they differed between nuget restore and dotnet restore.